tests/raft_scenarios_runner.py

```diff
@@ -139,7 +142,10 @@ def expand_files(files):
     ostream = sys.stdout
 
     # Create consensus-specific output directory
-    output_dir = os.path.join("consensus")
+    if args.output is None:
+        output_dir = os.path.join("consensus")
+    else:
+        output_dir = args.output
```

Comment:
```suggestion
```

Together with the change above? Shorter and a little more discoverable for the user?